packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/recursivelyReadFolderStructureWith.and.and.and..st

```diff
@@ -0,0 +1,17 @@
+read file
+recursivelyReadFolderStructureWith: maildirDirectory and: directoryName and: currentFolder and: anEndpoint
```

Comment:
I'd argue the distinction between `FileDirectory` and `ICFolder` should be more pronounced